### PR TITLE
Get started on helpdesk blog post

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
@netlify /blog/2024/helpdesk

### CODE REVIEW

File: sites/main-site/src/content/blog/2024/helpdesk.mdx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+pubDate: 2024-10-07T16:14:00+01:00
+authors:
+    - 'FriederikeHanssen'
+label:
+    - 'community'
+---
+
+# nf-core Helpdesk
+
+We are restarting nf-core office hours under a new name: *nf-core Helpdesk*.
```
Comment (mashehu) [inline sites/main-site/src/content/blog/2024/helpdesk.mdx]:
```suggestion
We are restarting nf-core office hours under a new name: *nf-core helpdesk*.
```

File: sites/main-site/src/content/blog/2024/helpdesk.mdx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+- [Gather Town](https://nf-co.re/join#gather-town)
+- [Slack: #helpdesk]()
+
+:::
+
+During the summer we trialed the [nf-core office hours](https://nf-co.re/blog/2024/office_hours). After gathering feedback, we decided to continue with them under a new name and
+broader scope: **nf-core Helpdesk**
+
+In these sessions, some of the nf-core maintainer team and other seasoned community members will be available to engage in discussions about your code, running pipelines, writing configs, or anything else related to nf-core and Nextflow.
+They will hang out in Gather during designated hours every week, ready to provide guidance and insights or just chat about your favorite pipeline.
```
Comment (mashehu) [inline sites/main-site/src/content/blog/2024/helpdesk.mdx:40]:
```suggestion
They will hang out in Gather during designated hours every week, ready to provide guidance and insights or just chat about your favourite pipeline.
```
🇬🇧 
(impressed by your restraint to not give an example for a favorite pipeline 😉 )

File: sites/main-site/src/content/blog/2024/helpdesk.mdx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+Helpdesk sessions won't be structured.
+
+Some example of things that might be discussed include:
```
Comment (mashehu) [inline sites/main-site/src/content/blog/2024/helpdesk.mdx:44]:
```suggestion
Things that might be discussed include:
```

File: sites/main-site/src/content/blog/2024/helpdesk.mdx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+-   A specific concept doesn't make sense and you'd like to talk it over
+-   You're working on a component or pipeline, you're stuck, and you'd like to get unstuck
+-   You'd like to swap code reviews with someone to get your PR merged
+-   You'd like to write an institution config and like some insights
+-   You can't figure out how to run a pipeline
+-   You want to hang out with other pipeline enthusiasts and get to know the faces behind the slack handles
```
Comment (mashehu) [inline sites/main-site/src/content/blog/2024/helpdesk.mdx:51]:
```suggestion
-   You want to hang out with other Nextflow enthusiasts and get to know the faces behind the slack handles
```

File: sites/main-site/src/content/blog/2024/helpdesk.mdx [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+
+The Helpdesk hours will take place in [Gather](https://gather.town/), an online collaborate platform, kind of like Pokemon-meets-Zoom. You can join by following the Gather Town line [here](https://nf-co.re/join#gather-town).
```
Comment (mashehu) [inline sites/main-site/src/content/blog/2024/helpdesk.mdx:53]:
```suggestion
The Helpdesk hours will take place in [Gather](https://gather.town/), an online collaborate platform, kind of like Pokemon-meets-Zoom. You can join by following the Gather Town link [here](https://nf-co.re/join#gather-town).
```

File: sites/main-site/src/content/blog/2024/helpdesk.mdx [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+For more information, see the [Gather Town bytesize talk](/events/2022/bytesize-37-gathertown).
+
+Please join the `#helpdesk` channel on the nf-core Slack to get reminders when sessions are about to start,
```
Comment (mashehu) [inline sites/main-site/src/content/blog/2024/helpdesk.mdx:56]: link to channel missing

**Review by mashehu — APPROVED** [review]

File: sites/main-site/src/content/blog/2024/helpdesk.mdx [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+**When**:
+
+-   Tuesdays, 2 PM Eastern Time
+-   Thursdays, 10 AM Central European Summer Time
+
+**Where**:
+
+-   [Gather Town](https://nf-co.re/join#gather-town)
+-   [Slack: #weekly-helpdesk](https://nfcore.slack.com/channels/weekly-helpdesk)
```
Comment (ewels) [inline sites/main-site/src/content/blog/2024/helpdesk.mdx:30]:
```suggestion
-   [Slack: `#weekly-helpdesk`](https://nfcore.slack.com/channels/weekly-helpdesk)
```

File: sites/main-site/src/content/blog/2024/helpdesk.mdx [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+    end: 2024-10-28T16:14:00+01:00
+pubDate: 2024-10-28T16:14:00+01:00
+authors:
+    - 'FriederikeHanssen'
+label:
+    - 'community'
+---
+
+We are restarting nf-core office hours under a new name and broader scope: _nf-core weekly Helpdesk_ on November 5th.
+These are weekly drop-in sessions designed for nf-core users and developers seeking an opportunity to connect, chat, and work together.
```
Comment (ewels) [inline sites/main-site/src/content/blog/2024/helpdesk.mdx:18]:
```suggestion
These are weekly virtual drop-in sessions designed for nf-core users and developers seeking an opportunity to connect, chat, and work together.
```

**Review by ewels — APPROVED** [review]

File: sites/main-site/src/content/blog/2024/helpdesk.mdx [hunk 9]
```diff
@@ -0,0 +1,55 @@
+---
+title: 'Office hours become Helpdesk'
```
Comment (ewels) [inline sites/main-site/src/content/blog/2024/helpdesk.mdx:2]:
```suggestion
title: 'Announcing: Weekly Helpdesk'
```

File: sites/main-site/src/content/blog/2024/helpdesk.mdx [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+subtitle: 'A weekly drop-in sessions to connect and work together.'
+headerImage: 'https://images.unsplash.com/photo-1545239351-ef35f43d514b'
+headerImageAlt: 'Person using macbook pro and holding cappuccino'
+announcement:
+    text: 'Office hours become Helpdesk'
```
Comment (ewels) [inline sites/main-site/src/content/blog/2024/helpdesk.mdx:7]:
```suggestion
    text: 'Announcing: Weekly Helpdesk'
```

**Review by ewels — APPROVED** [review]

File: sites/main-site/src/content/blog/2024/helpdesk.mdx [hunk 11]
```diff
@@ -0,0 +1,55 @@
+---
+title: 'Announcing: Weekly Helpdesk'
+subtitle: 'A weekly drop-in sessions to connect and work together.'
+headerImage: 'https://images.unsplash.com/photo-1545239351-ef35f43d514b'
+headerImageAlt: 'Person using macbook pro and holding cappuccino'
+announcement:
+    text: 'Announcing: Weekly Helpdesk'
+    start: 2024-10-28T16:14:00+01:00
+    end: 2024-10-28T16:14:00+01:00
+pubDate: 2024-10-28T16:14:00+01:00
```
Comment (ewels) [inline sites/main-site/src/content/blog/2024/helpdesk.mdx:10]: Publishing now? Or start of week / after my nf-core talk at the summit?

After my talk at summit times:
```suggestion
    start: 2024-10-31T16:18:00+01:00
    end: 2024-10-31T16:18:00+01:00
pubDate: 2024-10-31T16:18:00+01:00
```

**Pull request merged**